backend/python/app/services/interfaces/file_storage_service.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC, abstractmethod
+
+
+class IFileStorageService(ABC):
+    """
+    FileStorageService interface for handling blob storage
+    """
+
+    @abstractmethod
+    def get_file(self, file_name):
```

Comment:
Update signature with `expiration_time` please, and document that the expected type is a `timedelta`